.github/workflows/tests.yml

```diff
@@ -41,6 +41,24 @@ jobs:
         run: pip install -r ./api/requirements-dev.txt
       - name: Run test suite
         run: make caluma-lint
+  ember-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 10
+      - name: Install dependencies
+        run: yarn install
+        working-directory: ./ember
+      - name: Set UID
+        run: echo "UID=$(id --user)" > .env
+      - name: Build the containers
+        run: docker-compose up -d --build ember
+      - name: Lint the code
+        run: make ember-lint
+      - name: Run test suite
+        run: make ember-test
```

Comment:
What is the reason for starting the container here for testing instead of just locally?

Reply:
Multiple reasons:

 - it's the same workflow as in dev
 - the dockerfile is used and thus validated
 - the commands from the Makefile are used and thus validated

Reply:
Hmm okay, but isn't this creating just overhead in the testing ci? And im not sure but i think most people still dev with a local `yarn ember serve` (or at least i do).

Reply:
Changed. I've used the job from `caluma-portal-demo`. Is it really necessary to run the tests twice? This takes ages to finish...